BuildPackages.cmd

```diff
@@ -13,7 +13,14 @@ call :BuildNuGetPackage Sarif.Converters %Version% || goto :ExitFailed
 
 ::Build pre-release packages
 call :BuildNuGetPackage Sarif.Driver    %Version%-beta || goto :ExitFailed
-call :BuildNuGetPackage Sarif.Multitool %Version%-beta || goto :ExitFailed
+
+::Build Multitool package from nuspec
+echo .
+echo Building Sarif.Multitool package...
+echo .nuget\NuGet.exe pack .\src\Nuget\Sarif.Multitool.nuspec -Symbols -Properties configuration=%Configuration%;version=%Version% -Suffix beta -Verbosity Quiet -BasePath .\ -OutputDirectory .\bld\bin\Nuget
```

Comment:
I'm always uncomfortable with a hard-coded command line echo. It's so easy to get out of sync. Do we need it?

Reply:
I'm just following the existing practice. It's kind of nice to see the command line that was executed.

Reply:
I agree it's nice to see the command line. Another way to to that, though, would be to turn `echo` on just before you issue the command line, and turn it off afterwards. That would guarantee that you saw exactly what was issued, and reduce the amount of duplicate code in the script.

Reply:
OK will remove it